.github/workflows/update-lockfile.yml

```
@@ -1,43 +1,43 @@
+                  fetch-depth: 1
+
+            - name: Configure Git
+              uses: DanTheMan827/config-git-user-action@v1
+
+            - name: Setup Bun
+              uses: oven-sh/setup-bun@v1
+              with:
+                  bun-version: latest
+
+            - name: Cache dependencies
+              uses: actions/cache@v3
```

actionlint (1.7.11)

[error] the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)